synphot/utils.py

```diff
@@ -252,14 +252,20 @@ def merge_wavelengths(waveset1, waveset2, threshold=1e-12):
     return out_wavelengths
 
 
-def download_data(cdbs_root, verbose=True, dry_run=False):
-    """Download CDBS data files to given root directory.
+def download_data(path_root=None, verbose=True, dry_run=False):
```

Comment:
I am reverting the code to make `path_root` optional. It changes the default behavior and might confuse users.